tests/FamilyHubs.RequestForSupport.UnitTests/RclTemp/FamilyHubsUi/Configure/FamilyHubsUiOptionsValidationTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using FamilyHubs.RequestForSupport.UnitTests.RclTemp.FamilyHubsUi.Configure.Helpers;
+using FluentAssertions;
+using Microsoft.Extensions.Options;
+using RclTemp.FamilyHubsUi.Options.Configure;
+
+namespace FamilyHubs.RequestForSupport.UnitTests.RclTemp.FamilyHubsUi.Configure;
+
+public class FamilyHubsUiOptionsValidationTests : FamilyHubsUiOptionsTestBase
+{
+    public FamilyHubsUiOptionsValidation FamilyHubsUiOptionsValidation { get; set; }
+
+    public FamilyHubsUiOptionsValidationTests()
+    {
+        FamilyHubsUiOptionsValidation = new FamilyHubsUiOptionsValidation();
+    }
+
+    [Fact]
+    public void Validate_NoValidationErrorsTest()
+    {
+        // act
+        var result = FamilyHubsUiOptionsValidation.Validate(null, FamilyHubsUiOptions);
+
+        Assert.Equal(ValidateOptionsResult.Success, result);
+    }
+
+    [Theory]
+    [InlineData(null)]
+    [InlineData("h ttp://example.com")]
```

Comment:
Is the space between the h & p in h ttp://example.com intentional?

Reply:
Yes, it's a test that should cause the validation to fail.